source/layout.erb

```diff
@@ -122,6 +122,27 @@
         ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
         var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
       })();
+
+     $('#edit-page').on('click', function(event) {
+         event.preventDefault();
+         var editPageUrl = event.target.getAttribute('href')
+
+         var localPagePath = window.location.pathname.substring(0, window.location.pathname.length - 1)
```

Comment:
This doesn't work for URLs like http://guides.emberjs.com/v2.0.0/ and http://guides.emberjs.com/v2.0.0/object-model/, since they remove `index` from the end. Maybe you can just replace https://github.com/emberjs/guides/edit/master with https://api.github.com/repos/emberjs/guides/contents/source from `event.target.getAttribute('href')`.